libs/shared/ebpf_serialize.c

```diff
@@ -32,6 +32,10 @@ typedef struct _ebpf_serialized_helper_function_prototype
     uint32_t helper_id;
     ebpf_return_type_t return_type;
     ebpf_argument_type_t arguments[5];
+    struct
+    {
+        bool reallocate_packet : 1;
+    } flags;
```

Comment:
Use a named typedef in ebpf_program_types.h and then use that type here for flags rather than repeating each bit definition.